src/Components/Display-feed/DisplayFeed.css

```diff
@@ -1,27 +1,20 @@
 .display-feed {
     position: relative;
     margin-bottom: 7em;
+    width: 90%;
```

Comment:
Makes it uncentered in the smallest common phone view but we can readd later if necessary